### DISCUSSION TITLE
About submitting the abstract in conference

### DISCUSSION THREAD

**Discussion by tnzo12** [nlmixrdevelopment/nlmixr#527] [Q&A] [answered]
Hello sir

I'm a student in Lab of Clinical Pharmacy, Chungnam National University, Korea

I was working on TDM platform which uses nlmixr and I hope to submit an abstract about it to the PAGE 2021

Wonder if I need a permission from package contributors, or do I need to mention authors who's related to this package

Take care !

**Comment by mattfidler** [comment, marked as answer]
Typically you cite one or all of our articles:
 - https://doi.org/10.1002/psp4.12445
 - https://doi.org/10.1002/psp4.12471
 - https://doi.org/10.1002/psp4.12618

**Reply by tnzo12** [reply]
Thank you so much !
